spec/javascripts/Users/permissions.spec.js

```diff
@@ -281,11 +281,14 @@ describe('ServiceAccess', function () {
     let state = {}
     let node = () => render(<ServiceAccess/>, state)
 
+    // all services are enabled, so individual checkboxes are disabled
     expect(node()).toContainElement(disabled)
 
+    // only some (or none) services are enabled, so nothing is disabled
     state.admin_sections = ['services']
-    expect(node()).toContainElement(disabled)
+    expect(node()).not.toContainElement(disabled)
```

Comment:
To be honest, not sure why the test worked previously, but the current behavior is desired:
if there is a `services` section, it means that there is either an empty list, or a list of existing services. NO `service` section means that all current and future APIs are enabled.

Reply:
I'd say you extract individual blocks for each assertion instead of adding comments. For instance `it('renders disabled when all services are enabled`).